babbage.nomad

```diff
@@ -101,8 +106,15 @@ job "babbage" {
   group "publishing" {
     count = "{{PUBLISHING_TASK_COUNT}}"
 
-    constraint {
-      distinct_hosts = true
+    spread {
+      attribute = "${node.unique.id}"
+      weight    = 100
```

Comment:
would have thought this would have less weight than AZ?

Reply:
perhaps more 🤔

Reply:
Thats not how spread works, they are AND'd together .. such that they are equally spread by availability zone and then equally spread by unique host id's (or the other way around).

Reply:
hmm, that rings a bell

I was (mistakenly) thinking: what's the point in `weight` then?
i.e. to say "across AZ" is more/less important than "across hosts"

but it's more "I want 70% of instances when attribute==`X`"

sounds legit

Reply:
ah, no, looking at the spec, I think I was right, originally

Reply:
`percentage` is what my last-but-one comment referred to